src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -436,9 +436,8 @@ private static LfBus getLfBus(Terminal terminal, LfNetwork lfNetwork, boolean br
         return bus != null ? lfNetwork.getBusById(bus.getId()) : null;
     }
 
-    private static LfNetwork create(MutableInt num, List<Bus> buses, List<Switch> switches, LfNetworkParameters parameters, Reporter reporter) {
-        LfNetwork lfNetwork = new LfNetwork(num.getValue(), parameters.getSlackBusSelector());
-        num.increment();
+    private static LfNetwork create(Integer numCC, Integer numSC, List<Bus> buses, List<Switch> switches, LfNetworkParameters parameters, Reporter reporter) {
```

Comment:
Why Integer?

Reply:
Probably comes from the busesByCc Pair boxing, but no need to keep it. Corrected.